nuxt.config.js

```diff
@@ -92,8 +97,8 @@ export default {
   },
 
   router: {
-    middleware: 'commonHeaderRedirect',
```

Comment:
I couldn't find a need for this.

Reply:
This middleware feels like it would be a good place to redirect bcsc/idir/bceid routes to the respective signin URLs, but alas I couldn't get that working without errors.